docs/2.6.5.md

```diff
@@ -0,0 +1,44 @@
+# ProxySQL 2.6.5 and ProxySQL admin tools (2024-09-12)
+
+[Download](https://www.percona.com/download-proxysql)<br>
+[Install](install-v2.md)<br>
+[View the GitHub repository](https://github.com/percona/proxysql-admin-tool)
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator to solve complex replication topology issues.
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL.
+
+The [ProxySQL Admin (proxysql-admin)](proxysql-admin-tool-v2-config.md) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1](proxysql-v1.md) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2](proxysql-admin-tool-functions.md) supports Percona XtraDB Cluster without custom scripts.
+
+The [ProxySQL 2 and pxc_scheduler_handler](psh-overview.md) tool can automatically perform a failover due to node failures, service degradation, or maintenance. The pxc_scheduler_handler tool has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
+
+## Release highlights
+
+Compared to the previous [ProxySQL 2.6.3 release], this release includes all improvements and bug fixes available in [ProxySQL 2.6.4] and [ProxySQL 2.6.5].
+
+In this release, we are skipping ProxySQL 2.6.04. We are releasing ProxySQL 2.6.5, which is the latest ProxySQL release. This decision ensures that you receive the most up-to-date features and improvements.
+
+
+* Upgraded OpenSSL to version 3.3.1.
+
+* Added compatibility for Ubuntu 24 and Fedora 40.
+
+* The system now correctly handles the `USE` command when it ends with a semicolon. It removes any trailing semicolon automatically. Additionally, the system will now return an error if someone tries to use the USE command as part of a multi-statement query.
+
+* In versions 2.5.2 through 2.6.4, the `stats_mysql_query_digest.digest` feature generated lowercase output when it should have been uppercase.
+
+
+## Bug fixes
+
+* [PSQLADM-518](https://perconadev.atlassian.net/browse/PSQLADM-518): Adds a systemd unit that runs ProxySQL with the `--initial` flag, ensuring ProxySQL starts automatically on system boot and restarts after a system exit.
```

Comment:
```suggestion
* [PSQLADM-518](https://perconadev.atlassian.net/browse/PSQLADM-518): Adds a systemd unit `proxysql-initial.service` that can be used to re-initialize ProxySQL from the config file (after first startup the DB file is used instead of the config file). For usage, please refer to [this link](https://proxysql.com/documentation/getting-started/#:~:text=Reinitializing%20ProxySQL%20from%20the%20config%20file)
```